ocsp/responder.go

```diff
@@ -93,6 +93,17 @@ func NewResponder(source Source) *Responder {
 	}
 }
 
+func addCacheHeader(response http.ResponseWriter, maxAge *int) {
```

Comment:
This pattern of passing a pointer to an int is weird. How about instead take a regular int and:

```
if maxAge == 0 {
  ...max-age=0, no-cache
} else {
  ...max-age=%d
```